src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![feature(zero_one)]
```

Comment:
This is no bueno.  I recommend a custom trait that offers the same functionality, implemented on all the primitives.  It can be seamlessly swapped for `Zero` and `One` from an API perspective later if/when those traits are stabilized.